pages/table/simple.page.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+
+import { Button, Header, SpaceBetween, Table, TableProps } from '~components';
+import { setComponentMetrics } from '~components/internal/analytics';
+
+import ScreenshotArea from '../utils/screenshot-area';
+setComponentMetrics({
+  componentMounted: event => {
+    console.log(`${event.componentName}Rendered`, event);
```

Comment:
Is this necessary or was just for debugging?

Reply:
Just for debugging. We don't run any tests against this page and I wanted a page to share for validation.